cli/src/test/resources/ide-projects/build/project/settings/ide.properties

```diff
@@ -0,0 +1,6 @@
+JAVA_VERSION=17.0.10_7
+MAVEN_VERSION=3.9.6
+NODE_VERSION=v18.19.1
+NPM_VERSION=9.9.2
+GRADLE_VERSION=8.7
+MVN_BUILD_OPTS=clean install
```

Comment:
This is the default that is supposed to be build-in. Either remove or override with something else to test the overriding mechanism.

Reply:
Adjusted to clean compile